go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/viam-modules/viamrtsp
 
-go 1.23
+go 1.23.0
```

Comment:
any reason we're pinning the patch ver now?

Reply:
b/c this is what `go mod tidy` did